### ISSUE TITLE
Issue with editing/saving the database

### ISSUE THREAD

**Issue by machiav3lli** [moneymanagerex/android-money-manager-ex#1332]
I'm using the MMex FDroid app.
I noticed since a while that when I open a database, that database would be copied to Android/data/com.money.manager.ex/files/databases/ITSNAME.mmb and all the editing will happen then to this copy, resultig that the original one not being changed whatsoever. Other than that, the database path which points to this address couldn't be changed from within the app.
This issue causing the sync with syncthing to fail(original one being edited on linux, the copy being edited on android).

**Comment by MisterY** [comment]
Correct. This is how the process works when the content is retrieved through the Storage Framework. 
Any specific issue in regards to the app?

**Comment by MisterY** [comment]
The original, BTW, gets updated when the Synchronize option is triggered manually.

**Comment by machiav3lli** [comment]
but it looks like, that the sync option isn't available in the fdroid version of the app?! and export the DB is greyed out.

**Comment by MisterY** [comment]
The F-Droid version is the only available version of the app. Sync is available in the main menu, as before. 
Configuration and automation are not available yet.

**Comment by MisterY** [comment]
Export, I think, is a rudimentary code which was copying the database file from the app's folder to internal storage. This has been obsolete for a while.

**Comment by dl200010** [comment]
Syncthing does not work through the Storage Framework, either. The official app does not have Storage Framework It syncs files to the SD card/internal memory, so for all of MMEX knows it is coming directly from the SD card/internal memory. Another issue, quite related to this one, with it is the MMEX app does not write to the SD card/internal memory when editing the data. It keeps a copy in cache until you click synchronize. Why is the app "synchronizing" with the SD card/internal memory and why do you have to manually click it?

**Comment by ckagerer** [comment]
Same issue for me, for the following update.

```
Epic Games Launcher    EpicGames.EpicGamesLauncher    1.3.82.0 1.3.93.0  winget
```

I was able to force an update with this:

`winget upgrade EpicGames.EpicGamesLauncher --locale en-US`

The cause seems to lie in the manifesto. In which the entry InstallerLocal has been added.

![grafik](https://github.com/moneymanagerex/android-money-manager-ex/assets/4355402/19b8a968-f067-4417-b03e-f99a28a1d925)
